how-to/develop-with-buildd.rst

```diff
@@ -268,3 +292,88 @@ The buildd that you have just installed needs registering with Launchpad so that
     - Non-virtualized builders are much simpler: ``launchpad-buildd`` is cleaned synchronously over XML-RPC at the end of each build, and that's it.  If you use this, then you must be careful not to run any untrusted code on the builder (since a ``chroot`` or container escape could compromise the builder), and you'll need to uncheck "Require virtualized builders" on any PPAs, live file systems, recipes, etc. that you want to be allowed to build on this builder.
 
  4. After 30 seconds or so, the status of the builder on the builders page should be 'Idle'. This page does not auto-update, so refresh!
+
+Running a build on qastaging
+----------------------------
+
+We can use XML-RPC to interact also with qastaging/staging builders. 
+First of all, we should be able to ssh into our bastion and ssh into the ``launchpad-buildd-manager`` unit since it's the one that has the firewall rules to talk with builders.
+
+Then we should follow the same procedure to get the correct ``sha1sum`` of a backend image.
+
+.. code-block:: sh
+
+    $ ./manage-chroot -s bionic -a amd64 -i lxd get 
+    $ sha1sum livecd.ubuntu-base.lxd.tar.gz
+
+The command above will show us also the link from which the image is retrieved, let's save it since we will use it in our ``ensurepresent`` function.
```

Comment:
That makes no sense for someone who has not worked with this before. Do not say what you want to do, but why and what is the bigger picture of it.